keps/sig-cloud-provider/2395-removing-in-tree-cloud-providers/README.md

```diff
@@ -123,10 +123,11 @@ For alpha, the feature gates will be used for testing purposes. When enabled, te
 disabled by default.
 
 For beta, the feature gates will be on by default, meaning core components will disallow use of in-tree cloud providers. This will act as a warning for users to migrate to external components. Users may
-choose to continue using the in-tree provider by explicitly disabling the feature gates. Beta is targeted for v1.23 or v1.24.
+choose to continue using the in-tree provider by explicitly disabling the feature gates. Beta is targeted for v1.26.
```

Comment:
One thing we discussed in the last SIG call is that we need a majority of our CI signal converted to use CCM before we are confident to go Beta in v1.26. Can you add a note about that here or include it in the Beta graduation criteria (FYI this KEP is really old so it's using the really old format)

Reply:
added here.